### DISCUSSION TITLE
docker里面还是老版本呢

### DISCUSSION THREAD

**Discussion by resultzh** [helson-lin/ffandown#18] [General]
群晖docker里面搜不到v5版，还是老版本更新不了。
这个可以增加一个密码啊登录吗？

**Comment by resultzh** [comment]
是群晖里面搜到的那个

**Comment by helson-lin** [comment]
使用tag v5.0.4-beta​ 
群晖搜索 h55205l/ffandown
标签填 v5.0.4-beta​
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2023年12月31日 12:16 |
| 收件人 | helson-lin/ffandown ***@***.***> |
| 抄送人 | Subscribed ***@***.***> |
| 主题 | [helson-lin/ffandown] docker里面还是老版本呢 (Discussion #18) |

群晖docker里面搜不到v5版，还是老版本更新不了。
这个可以增加一个密码啊登录吗？
